docs/_docs/configuration.md

```diff
@@ -728,50 +725,6 @@ mutually exclusive.
 The various Markdown renderers supported by Jekyll sometimes have extra options
 available.
 
-### Redcarpet
```

Comment:
Should we leave a message to inform the deprecation here?

Reply:
I don't know if it makes sense to leave a deprecation message when support of Redcarpet will be removed (in 4.0). Maybe I should do another PR with deprecation warnings? And merge this PR to the master before 4.0?

Reply:
@nickskalkin Please don't bother adding deprecation messages to our *documentation*. Chances are that those messages will never be read..

Reply:
Changes to the documentation shouldn't be live before v4.0. I'd advise to commit those changes in a separate commit in the `docs-40` branch